app/forms/school_search_form.rb

```diff
@@ -3,23 +3,70 @@
 class SchoolSearchForm
   include ActiveModel::Model
 
-  attr_accessor :school_name, :location, :search_distance, :search_distance_unit, :characteristics, :partnership
+  CHARACTERISTICS = [OpenStruct.new(id: "pupil_premium_above_40", name: "Pupil premium above 40%"),
+                     OpenStruct.new(id: "top_20_remote_areas", name: "Located in remote area")].freeze
+
+  attr_accessor :school_name,
+                :location,
+                :search_distance,
+                :search_distance_unit,
+                :characteristics,
+                :partnership,
+                :local_authorities,
+                :networks,
+                :search_type,
+                :cohort_year,
+                :lead_provider_id,
+                :selected_cohort_id
 
   def find_schools(page)
-    schools = School.where("lower(name) LIKE ? OR
-                            lower(urn) LIKE ?",
-                           "%#{(school_name || '').downcase}%",
-                           "%#{(school_name || '').downcase}%")
-    .includes(:network, :lead_provider)
+    schools = School.where("schools.name ILIKE ? OR schools.urn ILIKE ?", "%#{school_name || ''}%", "%#{school_name || ''}%")
+                    .includes(:network, :lead_providers)
+
+    schools = schools.unpartnered(year) unless show_partnered?
+
+    schools = schools.partnered(year) if show_partnered?
```

Comment:
Ah, this is not going to work with what I have. The intention of this was that partnered schools would be excluded by default, and included with this parameter. Not that only partnered schools would be shown. I can fix this after it goes in though

Reply:
No worries - I can stick another attribute - any specific preference for the name?

Reply:
Done